libraries/AP_IOMCU/iofirmware/iofirmware.cpp

```diff
@@ -214,11 +298,11 @@ static void uart_lld_serve_tx_end_irq(hal_uart_driver *uart, uint32_t flags)
 
 void AP_IOMCU_FW::tx_dma_allocate(Shared_DMA *ctx)
 {
-    hal_uart_driver *uart = &UARTD2;
+    hal_uart_driver *uart = &HAL_IO_FMU_COMMS;
     chSysLock();
     if (uart->dmatx == nullptr) {
-        uart->dmatx = dmaStreamAllocI(STM32_UART_USART2_TX_DMA_STREAM,
-                                        STM32_UART_USART2_IRQ_PRIORITY,
+        uart->dmatx = dmaStreamAllocI(HAL_IO_FMU_COMMS_TX_DMA_STREAM,
+                                        12,  //IRQ Priority
```

Comment:
This is bypassing the config, thing it should use the define or put a define above

Reply:
yeah, I was trying to avoid adding yet another define that's going to remain untouched. Will add this to define as well.

Reply:
still pending?